index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CV</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+    crossorigin="anonymous">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU"
+    crossorigin="anonymous">
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+  <main>
+    <article class="main-info">
+      <div class="main-info__title">
+        <h1 class="initials">Kirill Lapkovskiy</h1>
+        <p class="specialty">Junior Javascript Developer</p>
+        <section class="summary">
+          <h2 class="visually-hidden">Summary</h2>
+          <p>
+            Hi, my name is Kirill.
+            I'm JavaScript Junior Developer in Artezio.
+            I develop client side of web applications and try using functional programming in my code.
+            My current goal in programming is to learn NodeJS and develop as fullstack developer.
+          </p>
+        </section>
+      </div>
+
+      <img class="photo" width="220" height="300" src="https://4.downloader.disk.yandex.by/preview/1b1f559f70996b238de82be0b016a9c7294dab03a998cf0861bf1f0fb003b6d3/inf/IKDeLiRCl0XmfflL4HXBolIS_nSlZhLq1cSolheY0NrlS45YVyvYR1cP2ewL_yr5axDFTTAIpZapf1HX2tQcQQ%3D%3D?uid=80283253&filename=1.jpg&disposition=inline&hash=&limit=0&content_type=image%2Fjpeg&tknv=v2&size=1903x969"
```

Comment:
It was better to move image in you repo, currently i can't see image because of 403 forbidden error from src URL.
![image](https://user-images.githubusercontent.com/14198624/47705276-1b0dc780-dc37-11e8-9dd8-aa4b2efee8a6.png)